### DISCUSSION TITLE
How can we show a Tooltip on Edge Label?

### DISCUSSION THREAD

**Discussion by kaushalendra-pandey** [xyflow/xyflow#1946] [Q&A]
<img width="337" alt="image" src="https://user-images.githubusercontent.com/67643722/156831394-1b8c9a85-c6bb-428a-b983-fdf75ea227d4.png">

I have React flow set up something like this. I want that when we hover on any of the labels, It shows a tooltip over it. Can anyone tell me how I can achieve this?

**Comment by Krithika3** [comment]
@kaushalendra-pandey were you able to achieve this?

**Reply by kaushalendra-pandey** [reply]
Nope.🙃
